website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -213,36 +200,84 @@ This property contains all the details of the edited rows. It has the following
 
-For example, if you updated the second row of a table. Now, when you bind this property into a text widget, the property displays the outcome as follows:&#x20;
+For example, if you update the second and fourth rows of a table, the `updatedRowIndices` property contains the value:
 
-```
+```javascript
 [ 
-1
+  1,
+  3
 ]
 ```
 
 #### updatedRow
 
-This property contains the details of the row that triggered the action (`onSubmit`, `onSave` or `onDiscard`). For example, if you bind this property into a text widget, you get an output something like this:
+This property contains the details of the row that triggered the **onSubmit**, **onSave** or **onDiscard** action. For example, if you bind this property into a text widget, you get an output something like this:
 
-```
+```javascript
 {
   "Name": "Updated Name",
   "Date": "Updated Date",
-  "Status ": "Updated Status",
-  "rowIndex": "Row Index"
+  "Status ": "Updated Status"
 }
 ```
 
 ### Events
 
+These properties can be used run code any time one of the events occurs. Be sure to take a look at the list of [supported actions](/reference/appsmith-framework/widget-actions/) that come built in to Appsmith.
+
 | Events        | Description                                                                                                            |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside. |
-| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                                                       |
-| **onDiscard** | Action that gets triggered when user clicks discard button for a cell or a new row.                                                       |
+| **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside of the cell. |
+| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                               |
+| **onDiscard** | Action that gets triggered when user clicks discard button for a cell or a new row.                                    |
+
+### Enable editing cells
+
+The column list on the Table property pane has a checkbox to make that column editable.
+
+Click the checkbox inside a column card makes all the cells in that column editable. The editable checkbox at the top of the list can be checked to make all the editable columns editable.
+
+Alternatively, A column can be editable by turning on `Editable` property inside the column's settings.
+
+![](</img/Editable_preoperty.png>)
+
+You can also use JS to control the `Editable` property. For example, you might use this feature to make only a subset of the cells editable:
+
+![](</img/Using_JS_in_Editable.png>)
```

Comment:
@danciaclara I have no idea why it seems to do this, but it goes back-and-forth on how the image renders. I've definitely seen the images super big / taking up the whole width of the page -- but currently they are just staying their true small size and behaving well. I'm not sure what makes them swap like they do.

![as_img_size](https://user-images.githubusercontent.com/51402465/204756243-028fe929-d4b9-4282-b123-2375a3b07df4.png)

Reply:
@spark-c did this get resolved when you specified width and height in the markdown?

Reply:
I believe the issue will be resolved with this; I can't test it for sure on my end because the issue was only showing in the deploy preview. However, I tested it on the deploy preview with the browser style inspector and it worked in that way.

I'll push these changes and we'll see what the new preview looks like! Will update

Reply:
It looks like the images are behaving now!